app/javascript/pages/ListingDirectory/DirectoryPage.tsx

```diff
@@ -76,9 +76,11 @@ export const getNumberString = (currencyNumber: number) =>
 
 export const getRangeString = (min: number, max: number, suffix?: string, prefix?: string) => {
   if (min && max && min !== max) {
+    const minString = `${prefix ?? ""}${getNumberString(min)}`
```

Comment:
this fixes an issue that our translation linter was having parsing this translation string